compatibility/README.md

```diff
@@ -59,6 +59,6 @@ Windows Phone 7.5
 
 
 Notes:
-*Outlook 2007-2019 only renders background images if it is coded in VML, there is an option in the generator to made both VML and HTML declaration for the background. [Read more](../generator-settings/VMLbackground.md)
+*Outlook 2007-2019 only renders background images if it is coded in VML, there is an option in the generator to made both VML and HTML declaration for the background. [Read more](../document-settings/VMLbackground.md)
 
- If you have any questions or run into any issues you can create an issue [here](https://github.com/EDMdesigner/editor-issues) 
+ If you have any questions or run into any issues you can create an issue [here](https://github.com/EDMdesigner/editor-issues/issues) 
```

Comment:
I just realized that the is leading to another repository. Shall we not handle generator issues in this repository?

Reply:
I agree I'll check this repository's settings about that and change every URL to point here.